src/DependencyInjection/Configuration.php

```diff
@@ -396,6 +396,8 @@ public function getConfigTreeBuilder()
                                 'bundles/sonataadmin/css/styles.css',
                                 'bundles/sonataadmin/css/layout.css',
                                 'bundles/sonataadmin/css/tree.css',
+
+                                'bundles/sonatacore/css/flashmessage.css',
```

Comment:
This file will always exist? No change in the `composer.json`?

Reply:
What do you mean? after the merge of the flashmessage stuff it will exists.

Reply:
In every CoreBundle version?

Reply:
Ah okay which version shall i use?

Reply:
We need to merge core bundle stuff, release it and then require this version as min version